### PR TITLE
#U9 feat: adds responsive header for desktop breakpoint

### PR DESCRIPTION
**Pull request by chrisostome01**
# PR Description

This pull request aims to add  responsive header for all screens

# Description of tasks that were expected to be completed

- [x] The header on the desktop should fit well as the design Figma we have, make sure it looks the same

# Description of tasks that were expected to be completed and that are not yet completed

- [x] none

# How this may be tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `npm i` to install all required dependencies 
2. run `npm run dev`
3. navigate to `localhost:3000 or 3001 `

# Screenshots (If appropriate/applicable)

# Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

File: components/sections/Header.jsx [hunk 1]
```diff
@@ -2,12 +2,12 @@ import CustomLink from "../ui/CustomLink";
 
 export default function Header() {
     return (
-        <section className="mt-6.6">
-            <div className="flex gap-5.5 flex-col text-2.5xl font-normal leading-7 ">
-                <span>
+        <section className="my-6.6  w-full">
+            <div className="flex gap-5.5 flex-col min-w-full text-2.5xl font-normal leading-7 lg:flex-row lg:w-[806px] lg:justify-between lg:text-2xl">
```
Comment (Yvad60) [inline components/sections/Header.jsx]: Remember to use `rem` units and also move  arbitrary values in the tailwind config

**Review by Yvad60 — CHANGES_REQUESTED** [review]
Good job @sezeranoJchrisostome fix one thing i mentioned about arbitrary values then this will be good to go!

File: components/sections/Header.jsx (hunk 1 above)
Comment (chrisostome01) [reply components/sections/Header.jsx]: thank you @Yvad60,

**Review by Yvad60 — APPROVED** [review]
Now it works😁!
Approved

**Review by serapieTuyishime — APPROVED** [review]

**Pull request merged**